UserTools/LoadWCSim/LoadWCSim.cpp

```diff
@@ -363,23 +370,21 @@ bool LoadWCSim::Execute(){
 	}
 	MCFile = WCSimEntry->GetCurrentFile()->GetName();
 	
+
 	MCHits->clear();
 	TDCData->clear();
 	MCNeutCap.clear();
 	MCNeutCapGammas.clear();
 	mrd_firstlayer=false;
-	mrd_lastlayer=false;
-	
+	mrd_lastlayer=false;	
+
 	triggers_event = WCSimEntry->wcsimrootevent->GetNumberOfEvents();
 	
-	int MaxEventNr = MCTriggernum+1;
-	if (!splitSubtriggers){
-		// we'll merge all subtriggers, so loop from 0 to triggers_event;
-		MCTriggernum = 0;
-		MaxEventNr = WCSimEntry->wcsimrootevent->GetNumberOfEvents();
-	} // else if we're splitting subtriggers, loop from [current MCTriggernum] to [current MCTriggernum+1]
-	
+	if (!splitSubtriggers) MCTriggernum = 0;
+	int MaxEventNr = (splitSubtriggers)? (MCTriggernum + 1) : (WCSimEntry->wcsimrootevent->GetNumberOfEvents());
```

Comment:
is there a reason to have replaced this code with a version that removes the explanatory comments?